cmd/keymasterd/storage.go

```diff
@@ -45,6 +54,7 @@ func (state *RuntimeState) expandStorageUrl() error {
 
 func initDB(state *RuntimeState) (err error) {
 	logger.Debugf(3, "Top of initDB")
+	state.Config.ProfileStorage.applyDefaults()
```

Comment:
I would only call this if the default values are zero.

Reply:
That's what applyDefaults() does: update the values if they are zero.
If you're suggesting adding a test here, that would make this code ugly.